images/ansible-operator/README.md

```diff
@@ -3,6 +3,6 @@ We build the base image using the Dockerfile, which validates the python require
 To update the requirements (`Pipfile` and `Pipfile.lock`) build and execute the image generated by `pipfile.Dockerfile` like so:
 
 1. docker build -f ./pipfile.Dockerfile -t pipfile-generator .
-2. docker run --rm -it -v .:/tmp/pip-airlock:Z pipfile-generator
+2. docker run --rm -it -v "$(pwd)":/tmp/pip-airlock pipfile-generator
```

Comment:
This should be reverted, since what is existing is likely to work on more systems.